WcaOnRails/app/assets/javascripts/application.js

```diff
@@ -240,8 +240,10 @@ Math.trunc = Math.trunc || function(x) {
 
 // Setting up bootstrap-table
 $(function() {
+  $('table[data-toggle="table"]').addClass('bootstrap-table');
```

Comment:
Haha, so I was actually suggesting getting rid of the `data-toggle` *entirely*, and initializing the bootstrap table plugin ourselves. As I understand it, the only purpose of that `data-toggle="table"` is to get this line of code to run for us: https://github.com/wenzhixin/bootstrap-table/blob/ccfabbe11fa7fbb494c916b10681a476f72cb2f2/src/bootstrap-table.js#L2995

```javascript
    // BOOTSTRAP TABLE INIT
    // =======================

    $(function () {
        $('[data-toggle="table"]').bootstrapTable();
    });
```